src/gui/properties/propertieswidget.cpp

```diff
@@ -76,7 +76,9 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     , m_ui(new Ui::PropertiesWidget())
 {
     m_ui->setupUi(this);
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)) && !defined(Q_OS_MACOS)
```

Comment:
Actually, this doesn't even look right. Qt 5.15 falls outside of this check.

Reply:
Did you test with Qt5? I'm under the impression that Qt5 need that autofill as well.

Reply:
Just tried with qt5. Not needed as well.

Reply:
>Just tried with qt5. Not needed as well. 

I disagree. This is on Windows with Qt5: ![screenshot](https://user-images.githubusercontent.com/9395168/188311609-4aa23fb1-d4d0-41b2-af56-ebf3ce2fadf5.png)
The color is unexpected/wrong.

The setting has been there for a long time and there are only one report on it. IMO this PR is more like a subjective tweak (or a Qt bug on macOS). Also I wonder what it will look like on bright theme on macOS? The problem might only manifest with dark theme.

Reply:
>Just tried with qt5. Not needed as well.

If this is true on macOS, then I suggest use the following (also assuming bright theme on macOS is OK with it):
```suggestion
#ifndef Q_OS_MACOS
```
Also please add `#include <QtGlobal>` on top of `#include <QClipboard>`

Reply:
Light theme on mac is ok with this. Had no idea about windows, was just questioning.